litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowPreview/styles.ts

```diff
@@ -18,7 +18,7 @@ const useStyles = makeStyles((theme: Theme) => ({
         cursor: 'pointer',
         fill: 'none',
         '& circle': {
-          fill: theme.palette.status.completed,
+          fill: theme.palette.status.experiment.completed,
         },
```

Comment:
As this is a workflow file so it should have colors from `status/workflow` and not `status/experiment`. May you please check with the design for the color and if required we may also do changes in the `status/workflow` for the required hex